### DISCUSSION TITLE
Show version number

### DISCUSSION THREAD

**Discussion by AnouckF** [microsoft-search/pnp-modern-search#3471] [Q&A] [answered]
Is there a possiblity to show the version number of a document in the Search results webpart? 

Using versio 4.9.3.0 of the Search webparts

**Comment by kasperbolarsen** [comment, marked as answer]
Oh yes, I would think UIVersionStringOWSTEXT should do the trick :-)

**Comment by AnouckF** [comment]
Thanks @kasperbolarsen It's working
